README.md

```diff
@@ -19,8 +19,6 @@ linux system).
 
 The dependencies to build the images are:
   - Yocto poky or later
-  - python3.4 or later (Installed on host machine)
-  - Lib python3-jsonschema (Installed on host machine)
```

Comment:
he is guaranteeing that the Yocto python3 provides the jsonschema (the python3-jsonschema-native), so you don't need to have it in your ubuntu/fedora/archlinux that is the host.

@mythi can you confirm my rationale is right?